### PR TITLE
changes to adress ROI setting logic

### PR DESCRIPTION
**Pull request by jontaylor010**
changes to fix treatment of the background ROI. Fixes reading roi data from file and logic for background options in peak tab. output refl data should be check for consistency and correctness

### CODE REVIEW

File: reflectivity_ui/interfaces/data_handling/data_info.py [hunk 1]
```diff
@@ -255,18 +258,18 @@ def determine_data_type(self, ws):
...
-                bck_range = copy.copy(self.roi_background)
+            # if not self.roi_background == [0, 0]:
+            #    bck_range = copy.copy(self.roi_background)
 
         # Background
         if self.use_tight_bck:
             bck_range = [int(max(0.0, peak[0] - self.bck_offset)), int(min(NX_PIXELS, peak[1] + self.bck_offset))]
-        elif self.use_roi_bck:
-            bck_range = [int(max(0.0, peak[0] - 2 * self.bck_offset)), int(max(0.0, peak[0] - self.bck_offset))]
+        # elif self.use_roi_bck:
```
Comment (jmborr) [inline reflectivity_ui/interfaces/data_handling/data_info.py:271]: same

**Review by jmborr — CHANGES_REQUESTED** [review]
The code looks good except for those bunch of commented lines that must go away.

Are there any instructions to carry out a manual test  or is this left to the CIS as part of the _AcceptanceTest_ phase?

File: reflectivity_ui/interfaces/data_handling/data_info.py [hunk 2]
```diff
@@ -255,18 +258,18 @@ def determine_data_type(self, ws):
             peak = copy.copy(self.roi_peak)
             if not self.roi_low_res == [0, 0]:
                 low_res = copy.copy(self.roi_low_res)
-            if not self.roi_background == [0, 0]:
-                bck_range = copy.copy(self.roi_background)
+            # if not self.roi_background == [0, 0]:
```
[the remark this answers is not in the document]
Comment (jontaylor010) [reply reflectivity_ui/interfaces/data_handling/data_info.py]: fixed with update

File: reflectivity_ui/interfaces/data_handling/data_info.py [hunk 3]
```diff
@@ -255,18 +258,18 @@ def determine_data_type(self, ws):
...
+            #    bck_range = copy.copy(self.roi_background)
         elif self.use_roi and self.update_peak_range and not self.roi_peak == [0, 0]:
             logging.info("Using fit peak range: [%s %s]" % (peak[0], peak[1]))
-            if not self.roi_background == [0, 0]:
-                bck_range = copy.copy(self.roi_background)
+            # if not self.roi_background == [0, 0]:
```
[the remark this answers is not in the document]
Comment (jontaylor010) [reply reflectivity_ui/interfaces/data_handling/data_info.py]: fixed with update

File: reflectivity_ui/interfaces/data_handling/data_info.py (hunk 1 above)
Comment (jontaylor010) [reply reflectivity_ui/interfaces/data_handling/data_info.py]: fixed with update

**Review by jmborr — APPROVED** [review]
:shipit:

**Pull request merged**